_docs/install/prerequisites/clusterdiscovery.md

```diff
@@ -7,56 +7,51 @@ module: install/prerequisites/clusterdiscovery
 
 # Cluster discovery
 
-StorageOS nodes must be able to contact each other over the network.  By default, it is assumed that this is the node's first non-loopback address.
+StorageOS nodes need to know the exact cluster size and peers to connect to
+during start up. This enables nodes to contact each other over the network.
 
-## Setting the IP address manually
+## Setting the IP address
 
-In some cases (such as with Vagrant installations), this will not be appropriate and it will need to be set manually.
-
-Use `ip a` to list available ip addresses, then set the `ADVERTISE_IP` environment variable on each node to configure StorageOS to use a
-specific address:
-
-```bash
-export ADVERTISE_IP=172.28.128.3
-```
-
-To view the status of a cluster, run `storageos node ls`:
+By default, a node's IP address is assumed to be the first non-loopback address.
+To override this (eg. for Vagrant installations), set the `ADVERTISE_IP`
+environment variable on each node to configure StorageOS to use a specific
+address:
 
 ```bash
-$ storageos node ls
-
-NAME                  ADDRESS             HEALTH              SCHEDULER           VOLUMES             TOTAL               USED                VERSION             LABELS
-vol-test-2gb-lon101   46.101.50.155       Healthy 2 days      true                M: 0, R: 2          77.43GiB            5.66%               0.7 (00ab7b3 rev)
-vol-test-2gb-lon102   46.101.50.231       Healthy 2 days      false               M: 1, R: 0          38.71GiB            5.90%               0.7 (00ab7b3 rev)
-vol-test-2gb-lon103   46.101.51.16        Healthy 2 days      false               M: 1, R: 1          77.43GiB            5.61%               0.7 (00ab7b3 rev)
+ADVERTISE_IP=172.28.128.3
 ```
 
-<!--
-A StorageOS cluster needs to know the exact cluster size and peers to connect to
-during start up.
+## Option 1: Specify cluster size
 
+The first option is to use the StorageOS discovery service, which is a public `etcd` service.
 
-## Cluster discovery
+Specify the expected size of the cluster using the [StorageOS CLI]({%link
+_docs/reference/cli/cluster.md %}):
```

Comment:
Think we need to be clear that this is not the size of the StorageOS cluster (that users need to know that up front), but the size of the consensus group to use, which can be 1, 3, 5, or 7 nodes (with 3 or more required for production).  I wonder if using cluster here is confusing?